packages/perseus-editor/src/widgets/free-response-editor.tsx

```diff
@@ -40,6 +44,15 @@ class FreeResponseEditor extends React.Component<Props> {
         return warnings;
     }
 
+    handleUpdateCharacterLimit: ChangeEventHandler<HTMLInputElement> = (e) => {
+        const val = parseInt(e.target.value);
+        if (isNaN(val)) {
+            return;
+        }
+
+        this.props.onChange({characterLimit: Math.max(1, val)});
```

Comment:
I think this makes sense from the technical stand-point. I wonder if we also need some kind of linter that warns for suspiciously low/high numbers - like if someone meant to write 100 and accidentally wrote 1...yeah that's _technically_ fine but do we really want a free response widget that only accepts one character?

Reply:
That's an interesting idea. I'm curious to see what character limits the content team ends up creating. Do you know if we get insight into this kind of data?